### PR TITLE
docs: source/mentorprogrammeapplication.md, their -> here

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**

### CODE REVIEW

File: source/mentorprogrammeapplication.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: MentorProgrammeApplication
+title: MentorProgrammeApplicationhowever mentors are not their to teach you
```
Comment (rlenferink) [inline source/mentorprogrammeapplication.md:2]: Can you update this to a valid title again? It seems a copy/paste mistake was made.
```suggestion
title: Mentor Programme Application
```

File: source/mentorprogrammeapplication.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ hope to gain from it and the mentor programme
  * A brief CV of your skills
 
 Note, we do not expect people to come to our mentor programme with a full
-armoury of skills, however mentors are not their to teach you only to guide
+armoury of skills, however mentors are not here to teach you, but only to guide
```
Comment (rlenferink) [inline source/mentorprogrammeapplication.md:46]: Disclaimer: non-native English here. Updating `their` to `there` makes most sense IMO. Any input here @cottage14 ?

```suggestion
armoury of skills, however mentors are not there to teach you, but only to guide
```

**Review by adityasharma7 — APPROVED** [review]

**Comment by Abdur-rahmaanJ** [comment]
Yay!

**Comment by rlenferink** [comment]
> Yay!

Thanks for contributing @Abdur-rahmaanJ !

**Pull request merged**